File: src/main/java/ru/serdtsev/homemoney/moneyoper/model/MoneyOperStatus.java
package ru.serdtsev.homemoney.moneyoper.model;

public enum MoneyOperStatus {
  // todo Порефакторить.
  /**
   * в ожидании
   */
  pending,
  /**
   * ожидает повтора
   */
  recurrence,
  /**
   * выполнен
   */
  done,
  /**
   * выполнен, для повторяющихся операций
   */
  doneNew,
  /**
   * отменен, конечный статус
   */
  cancelled,
  /**
   * шаблон
   */
  template
}

File: src/main/kotlin/ru/serdtsev/homemoney/moneyoper/model/MoneyOperStatus.kt
package ru.serdtsev.homemoney.moneyoper.model

enum class MoneyOperStatus {
    // todo Порефакторить.
    /** в ожидании */
    pending,
    /** ожидает повтора */
    recurrence,
    /** выполнен */
    done,
    /** выполнен, для новых операций (промежуточный статус, который транслируется либо в done, либо в pending) */
    doneNew,
    /** отменен, конечный статус */
    cancelled,
    /** шаблон */
    template
}